go.mod

```diff
@@ -90,4 +83,8 @@ replace github.com/lightningnetwork/lnd/clock => ./clock
 
 replace git.schwanenlied.me/yawning/bsaes.git => github.com/Yawning/bsaes v0.0.0-20180720073208-c0276d75487e
 
+// Fix incompatibility of etcd go.mod package.
```

Comment:
We could get rid of this replace if we used the version `v0.5.0-alpha.5.0.20201125193152-8a03d2e9614b` directly above instead of `v3.4.14+incompatible`. Not sure if that would perhaps make it a bit easier in projects that depend on `lnd`? Because the replace directives will be ignored in dependent projects. But not really sure if it makes a big difference.

Reply:
I like the replace directive since it gives pointers to an active issue that we should keep an eye on. Happy to go whatever route you recommend though.